items.json

```diff
@@ -30822,30 +30822,30 @@
         "missileSpeed": 0,
         "stackAmount": 0,
         "length": 67,
-        "balance": 0.68,
-        "handling": 92,
+        "balance": 0.64,
+        "handling": 91,
         "bodyArmor": 0,
         "flags": [
           "MeleeWeapon"
         ],
         "thrustDamage": 0,
         "thrustDamageType": "Undefined",
         "thrustSpeed": 88,
-        "swingDamage": 26,
+        "swingDamage": 25,
         "swingDamageType": "Blunt",
-        "swingSpeed": 90
+        "swingSpeed": 89
       }
     ]
   },
   {
-    "id": "crpg_calradic_mace_v1_h0",
-    "baseId": "crpg_calradic_mace_v1",
+    "id": "crpg_calradic_mace_v2_h0",
+    "baseId": "crpg_calradic_mace_v2",
     "name": "Calradic Mace",
     "culture": "Empire",
     "type": "OneHandedWeapon",
-    "price": 6281,
-    "weight": 1.72,
-    "tier": 9.413677,
+    "price": 4069,
+    "weight": 1.81,
+    "tier": 7.35880756,
```

Comment:
speed reduction is OK but should be offset by an increase in damage to match tiers